Library/Homebrew/os/mac.rb

```diff
@@ -24,7 +24,9 @@ def locate tool
       # Give the name of the binary you look for as a string to this method
       # in order to get the full path back as a Pathname.
       (@locate ||= {}).fetch(tool) do |key|
-        @locate[key] = if File.executable?(path = "/usr/bin/#{tool}")
+        @locate[key] = if File.executable?(path = "#{Formula['cctools'].opt_prefix}/bin/#{tool}")
```

Comment:
Can do! This particular behavior is really only needed when we attempt to call `otool` and `install_name_tool` on unarchived bottles, so I could write a wrapper function like `MacOS.locate_cctools` or simply include this logic in each place.